source/common/runtime/runtime_features.cc

```diff
@@ -88,6 +88,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.strict_1xx_and_204_response_headers",
     "envoy.reloadable_features.tls_use_io_handle_bio",
     "envoy.reloadable_features.unify_grpc_handling",
+    "envoy.reloadable_features.use_apple_api_for_dns_lookups",
```

Comment:
Per discussion, startup_features?

Reply:
It didn't seem that you all settled on what to do about these back in the thread I linked above, and I couldn't find an update on that kind of runtime flag. But if you are onboard with the namespace and documentation, I am too.